packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -49,8 +49,47 @@ class FrameFlameChart extends CoreElement {
       ..overflow = 'hidden';
 
     enableDragScrolling(this);
+    element.onMouseMove.listen((MouseEvent e) => _handleMouseMove(e));
+    element.onMouseWheel.listen((WheelEvent e) => _handleMouseWheel(e));
   }
 
+  /// Target maximum microseconds per frame.
+  ///
+  /// 16,666 microseconds per frame will achieve a frame rate of 60 FPS.
+  static const double targetMicrosPerFrame = 1000 * 1000 / 60.0;
+
+  /// Pixels per microsecond in order to fit a single frame in 1500px.
+  ///
+  /// For the whole frame to fit in 1500px at this drawing ratio, the frame
+  /// duration must be [targetMicrosPerFrame] or less. 1500px is arbitrary.
+  static const double pixelsPerMicro = 1500 / targetMicrosPerFrame;
+
+  num _zoomLevel = 1;
+  num _minZoomLevel;
+
+  // Throttling the zoomUnit makes the scrolling smoother.
+  num get _zoomUnit {
+    if (_zoomLevel <= 1) {
```

Comment:
nit: can this be a simple function of the zoom level.

maybe this could be zoomSpeedMultiplier or some similar name?

Reply:
done.